### PR TITLE
PLAT-104442: Not correctly providing theme ResBundle constant for all themes

### PR DESCRIPTION
**Pull request by JayCanuck**
Some old code in our Jest setup was only handling Moonstone constants correctly (setting `ILIB_MOONSTONE_PATH`).

There was a bug for themed apps where we were setting `<THEME>` global instead of `ILIB_<THEME>_PATH` like we were meant to. This fixes that.

Additionally, to provide generalized support for in-theme usage the local resource resbundle path is added as `ILIB_<NAME>_PATH` to cover in-theme testing and some edge cases.

Enact-DCO-1.0-Signed-off-by: Jason Robitaille <jason.robitaille@lge.com>

### CODE REVIEW

File: config/jest/jest.config.js [hunk 1]
```diff
@@ -12,25 +12,26 @@ const fs = require('fs');
...
+	'_PATH';
+
 const iLibDirs = ['node_modules/@enact/i18n/ilib', 'node_modules/ilib', 'ilib'];
 const globals = {
 	__DEV__: true,
 	ILIB_BASE_PATH: iLibDirs.find(f => fs.existsSync(path.join(app.context, f))) || iLibDirs[1],
 	ILIB_RESOURCES_PATH: 'resources',
-	ILIB_CACHE_ID: new Date().getTime() + ''
+	ILIB_CACHE_ID: new Date().getTime() + '',
+	[rbConst(app.name.split('/').pop())]: 'resources'
```
Comment (ryanjduffy) [inline config/jest/jest.config.js:29]: I suspect `split` and `pop` are redundant here with the `basename` call in `rbConst`.

**Comment by ryanjduffy** [comment]
Update: We'll merge this into `develop` and cherry pick into a release branch for 2.x.
